Wolox/wlogger-ios
# login to our API and tests

## Sumary ##
**Trello card**: https://trello.com/c/sDtkzkJl/1-funcionalidad-log-in-con-google

Login to the API sending the G+ token and getting the trello token

## Test Plan ##
  
  * Open the app
  * Press the login button and check that the app does not crash
 
## Screenshots ##

No visual changes between the last pr and this one
